code/modules/research/xenobiology/crossbreeding/industrial.dm

```diff
@@ -117,6 +117,9 @@ Industrial extracts:
 
 /obj/item/slimecross/industrial/silver/process()
 	itempath = pick(list(get_random_food(), get_random_drink()))
+	if(istype(itempath, /obj/item/food))
+		var/obj/item/food/food = new itempath
+		food.food_flags |= FOOD_SILVER_SPAWNED
```

Comment:
Don't think `itempath` works for industrials. 

In fact, there's a perfect proc for you to override to do this in - `do_after_spawn(/obj/item/spawned)`